src/test/groovy/com/stormpath/tck/forgot/ChangePasswordIT.groovy

```diff
@@ -168,12 +176,18 @@ class ChangePasswordIT extends AbstractIT {
     @Test(groups=["v100", "html"])
     public void changeRedirectsToErrorUriForInvalidSptokenWhenPosting() throws Exception {
 
-        given()
+        saveCSRFAndCookies(ForgotRoute)
+
+        def req = given()
             .accept(ContentType.HTML)
             .contentType(ContentType.URLENC)
             .queryParam("sptoken", "NOTEVENCLOSETOVALID")
             .param("password", "foobar123!")
-        .when()
+            .param("confirmPassword","foobar123!")
```

Comment:
The above link is for `register`. @edjiang did point out that we decided to remove a password confirmation from the framework spec in the past. However, it's not currently specified one way or the other. It's not worth code changes in the Java integrations IMHO to remove the confirm password field for resetting your password. @omgitstom - would you please weigh in on this? It's holding up @edjiang's willingness to merge this PR.